pages/mesosphere/dcos/version-policy/index.md

```diff
@@ -366,13 +366,13 @@ The designations are as follows:
         <td><p style="text-align: center;">⚫</p></td>
     </tr>
     <tr>
-        <td>Elastic 2.7.x-6.8.1</td>
+        <td>Elastic 3.1.0-7.4.1</td>
```

Comment:
```suggestion
        <td>Elastic 2.7.x-6.8.1</td>
```

Reply:
Hmm, this is to keep the latest major version + latest major version -1 in the docs (i.e 7.X.X and 6.X.X) at any point of time.

Reply:
Done. Thanks for the suggestion!